routes/web.php

```diff
@@ -29,3 +34,10 @@
 });
 
 require __DIR__.'/auth.php';
+
+Route::get('brands', [UserController::class,'index'])->name('brands');
+Route::post('brand',[UserController::class,'store'])->name('store_brand');
+Route::get('edit/{id}', [UserController::class, 'edit'])->name('edit');
+Route::post('update/{id}', [UserController::class, 'update'])->name('update_brand');
```

Comment:
add in group

change URL and name also

Example
------------------

Route::controller(NotificationController::class)->name('notifications.')->group(function (): void {
    Route::get('fetch-notifications', 'fetchNotifications')->name('fetch');
    Route::post('mark-all-as-read', 'markAllAsRead')->name('mark_all_as_read');
});